### PR TITLE
Papa 7 - Recovered

### PR DESCRIPTION
**Pull request by cdobs**
Creates ingestion csvs for Papa 7 recovered moorings

### CODE REVIEW

File: GP02HYPM/GP02HYPM_R00007_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+mi.dataset.driver.vel3d_l.wfp.vel3d_l_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00007/instruments/GWFP_sn12997-01/0*/A*.DA?,GP02HYPM-WFP02-05-VEL3DL000,recovered_wfp,Available,
+#mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00007/instruments/GWFP_sn13104-03/0*/E*.DA?,GP02HYPM-WFP03-00-WFPENG000,recovered_wfp,Pending,Can't parse data. Redmine 12792
+mi.dataset.driver.flord_l_wfp.flord_l_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00007/instruments/GWFP_sn13104-03/0*/E*.DA?,GP02HYPM-WFP03-01-FLORDL000,recovered_wfp,Available,
+mi.dataset.driver.dosta_ln.wfp.dosta_ln_wfp_driver,/omc_data/whoi/OMC/GP02HYPM/R00007/instruments/GWFP_sn13104-03/0*/E*.DA?,GP02HYPM-WFP03-03-DOSTAL000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00007/instruments/GWFP_sn13104-03/0*/C*.DA?,GP02HYPM-WFP03-04-CTDPFL000,recovered_wfp,Available,
+mi.dataset.driver.vel3d_l.wfp.vel3d_l_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00007/instruments/GWFP_sn13104-03/0*/A*.DA?,GP02HYPM-WFP03-05-VEL3DL000,recovered_wfp,Available,
```
Comment (snwhite) [inline GP02HYPM/GP02HYPM_R00007_ingest.csv:16]: Looks good.

File: GP03FLMA/GP03FLMA_R00007_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMA/R00007/instruments/CTDMO_sn37-12389/SBE37*.hex,GP03FLMA-RIM01-02-CTDMOH051,recovered_inst,Available,
+mi.dataset.driver.sio_eng.sio.sio_eng_sio_recovered_driver,/omc_data/whoi/OMC/GP03FLMA/R00007/cg_data/SSIOC_sn*/STATUS/STA*.DAT,GP03FLMA-RIS01-00-SIOENG000,recovered_host,Available,
+mi.dataset.driver.dosta_abcdjm.sio.dosta_abcdjm_sio_recovered_driver,/omc_data/whoi/OMC/GP03FLMA/R00007/cg_data/SSIOC_sn*/DOSTA/DOS*.DAT,GP03FLMA-RIS01-03-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GP03FLMA/R00007/instruments/PHSEN_sn*/SAMI*.txt,GP03FLMA-RIS01-04-PHSENF000,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,/omc_data/whoi/OMC/GP03FLMA/R00007/cg_data/SSIOC_sn*/PHSEN/PHS*.DAT,GP03FLMA-RIS01-04-PHSENF000,recovered_host,Available,
+mi.dataset.driver.flort_dj.sio.flort_dj_sio_recovered_driver,/omc_data/whoi/OMC/GP03FLMA/R00007/cg_data/SSIOC_sn*/FLORT/FLO*.DAT,GP03FLMA-RIS01-05-FLORTD000,recovered_host,Available,
```
Comment (snwhite) [inline GP03FLMA/GP03FLMA_R00007_ingest.csv:34]: Looks good.

File: GP03FLMB/GP03FLMB_R00007_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00007/instruments/CTDMO_sn37-10228/SBE37*.hex,GP03FLMB-RIM01-02-CTDMOH071,recovered_inst,Available,
+mi.dataset.driver.sio_eng.sio.sio_eng_sio_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00007/cg_data/SSIOC_sn*/STATUS/STA*.DAT,GP03FLMB-RIS01-00-SIOENG000,recovered_host,Available,
+mi.dataset.driver.dosta_abcdjm.sio.dosta_abcdjm_sio_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00007/cg_data/SSIOC_sn*/DOSTA/DOS*.DAT,GP03FLMB-RIS01-03-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00007/instruments/PHSEN_sn*/SAMI*.txt,GP03FLMB-RIS01-04-PHSENF000,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00007/cg_data/SSIOC_sn*/PHSEN/PHS*.DAT,GP03FLMB-RIS01-04-PHSENF000,recovered_host,Available,
+mi.dataset.driver.flort_dj.sio.flort_dj_sio_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00007/cg_data/SSIOC_sn*/FLORT/FLO*.DAT,GP03FLMB-RIS01-05-FLORTD000,recovered_host,Available,
```
Comment (snwhite) [inline GP03FLMB/GP03FLMB_R00007_ingest.csv:34]: Looks good.

**Review by snwhite — APPROVED** [review]

File: GP02HYPM/GP02HYPM_R00007_ingest.csv (hunk 1 above)
Comment (IDCalTech) [reply GP02HYPM/GP02HYPM_R00007_ingest.csv:16]: Looks good

File: GP03FLMB/GP03FLMB_R00007_ingest.csv (hunk 3 above)
Comment (rtravis-whoi) [reply GP03FLMB/GP03FLMB_R00007_ingest.csv:34]: looks good

**Review by rtravis-whoi — APPROVED** [review]
I reviewed GP03FLMB_R00007_ingest.csv only - approve.

File: GP03FLMA/GP03FLMA_R00007_ingest.csv (hunk 2 above)
Comment (IDCalTech) [reply GP03FLMA/GP03FLMA_R00007_ingest.csv:34]: Looks good

**Review by IDCalTech — APPROVED** [review]
I reviewed  GP02HYPM/GP02HYPM_R00007_ingest.csv and GP03FLMA/GP03FLMA_R00007_ingest.csv, both look good.

**Pull request merged**